.github/workflows/release.yml

```diff
@@ -191,17 +193,19 @@ jobs:
 
                 # Fix the dependency
                 echo "  Patching $dep"
-                install_name_tool -change $dep @executable_path/../Frameworks/lib/$dep_file $file
+                install_name_tool -change $dep @executable_path/$rpath/lib/$dep_file $file
 
                 # Collect list of dependencies
                 process_list="$new_dep_file $process_list"
               fi
             done
 
             # Recursively process dependencies
-            for dep in $process_list; do
-              process_dependencies $target $destdir $dep
-            done
+            if [[ $recursive == "true" ]]; then
+              for dep in $process_list; do
+                process_dependencies $target $destdir $dep $rpath $recursive
+              done
+            fi
```

Comment:
By the way, what is recursive in there that this is actually needed? On Windows all files are flat and nothing else is needed. I do not see any hierarchy in the final structure.

Reply:
Do we have such cases though? I'd expect to simply copy all dylib files from GTK to the package like it is done on Windows with a single command, without any recursion and other complexity.

Reply:
@nazar-pc Yes! One example is `space-acres` depends on `libgtk-4.1`, `libgtk-4.1` depends on `libgobject-2.0.0`,  `libgdk_pixbuf-2.0.0`, and then `libgdk_pixbuf-2.0.0` depends on `libgio-2.0.0` ...

If a flat list is better, I can make a list of files to be copied (the output of CI is list of files required) instead of recursing processing. What do you think?

Reply:
If all the files are in the same place (which I expect is the case), then I expect something like this:
https://github.com/subspace/space-acres/blob/3db9689e6646f3fd3b6b3d550fb43c42f7ccfa8b/.github/workflows/release.yml#L177

Or a single loop that copies and fixes files individually instead of recursive functions, which are harder to follow. Since we don't need to include them explicitly like we do in Windows package, I think it is fine to not have a list, just copy all dynamic libraries that are in there.

Reply:
@nazar-pc I've created flat list of operations (copy and fix path), but then it quite long list and can't be put in 1 stage of github action (as it only maximum 21000 characters in 1 stage). What do you think? The list like this:

```
cp $BREW_PREFIX/opt/glib/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/gtk4/lib/libgtk-4.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/pango/lib/libpangocairo-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/pango/lib/libpango-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/harfbuzz/lib/libharfbuzz.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/gdk-pixbuf/lib/libgdk_pixbuf-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/cairo/lib/libcairo-gobject.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/cairo/lib/libcairo.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/graphene/lib/libgraphene-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/glib/lib/libgio-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/gettext/lib/libintl.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/pcre2/lib/libpcre2-8.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/Cellar/glib/2.80.2/lib/libgmodule-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/libpng/lib/libpng16.16.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/fontconfig/lib/libfontconfig.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/freetype/lib/libfreetype.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/libx11/lib/libX11.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/libxext/lib/libXext.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/libxrender/lib/libXrender.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/libxcb/lib/libxcb.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/libxcb/lib/libxcb-render.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/libxcb/lib/libxcb-shm.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/pixman/lib/libpixman-1.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/libxau/lib/libXau.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/libxdmcp/lib/libXdmcp.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/jpeg-turbo/lib/libjpeg.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/graphite2/lib/libgraphite2.3.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/fribidi/lib/libfribidi.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/Cellar/pango/1.52.2/lib/libpangoft2-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/libepoxy/lib/libepoxy.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/libtiff/lib/libtiff.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/cairo/lib/libcairo-script-interpreter.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/lzo/lib/liblzo2.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/zstd/lib/libzstd.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/opt/xz/lib/liblzma.5.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/Cellar/librsvg/2.58.1/lib/librsvg-2.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib
cp $BREW_PREFIX/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders

install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgobject-2.0.0.dylib @executable_path/../Resources/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/gtk4/lib/libgtk-4.1.dylib @executable_path/../Resources/lib/libgtk-4.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/pango/lib/libpangocairo-1.0.0.dylib @executable_path/../Resources/lib/libpangocairo-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/pango/lib/libpango-1.0.0.dylib @executable_path/../Resources/lib/libpango-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/harfbuzz/lib/libharfbuzz.0.dylib @executable_path/../Resources/lib/libharfbuzz.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/gdk-pixbuf/lib/libgdk_pixbuf-2.0.0.dylib @executable_path/../Resources/lib/libgdk_pixbuf-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/cairo/lib/libcairo-gobject.2.dylib @executable_path/../Resources/lib/libcairo-gobject.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/cairo/lib/libcairo.2.dylib @executable_path/../Resources/lib/libcairo.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/graphene/lib/libgraphene-1.0.0.dylib @executable_path/../Resources/lib/libgraphene-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgio-2.0.0.dylib @executable_path/../Resources/lib/libgio-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/gettext/lib/libintl.8.dylib @executable_path/../Resources/lib/libintl.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
install_name_tool -change $BREW_PREFIX/opt/gettext/lib/libintl.8.dylib @executable_path/../Resources/lib/libintl.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libglib-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/pcre2/lib/libpcre2-8.0.dylib @executable_path/../Resources/lib/libpcre2-8.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libglib-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/Cellar/glib/2.80.2/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgio-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/Cellar/glib/2.80.2/lib/libgobject-2.0.0.dylib @executable_path/../Resources/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgio-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/Cellar/glib/2.80.2/lib/libgmodule-2.0.0.dylib @executable_path/../Resources/lib/libgmodule-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgio-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/gettext/lib/libintl.8.dylib @executable_path/../Resources/lib/libintl.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgio-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/Cellar/glib/2.80.2/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgmodule-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/Cellar/glib/2.80.2/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgobject-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgobject-2.0.0.dylib @executable_path/../Resources/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgraphene-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgraphene-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/gettext/lib/libintl.8.dylib @executable_path/../Resources/lib/libintl.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgraphene-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/libpng/lib/libpng16.16.dylib @executable_path/../Resources/lib/libpng16.16.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo.2.dylib
install_name_tool -change $BREW_PREFIX/opt/fontconfig/lib/libfontconfig.1.dylib @executable_path/../Resources/lib/libfontconfig.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo.2.dylib
install_name_tool -change $BREW_PREFIX/opt/freetype/lib/libfreetype.6.dylib @executable_path/../Resources/lib/libfreetype.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libx11/lib/libX11.6.dylib @executable_path/../Resources/lib/libX11.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxext/lib/libXext.6.dylib @executable_path/../Resources/lib/libXext.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxrender/lib/libXrender.1.dylib @executable_path/../Resources/lib/libXrender.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxcb/lib/libxcb.1.dylib @executable_path/../Resources/lib/libxcb.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxcb/lib/libxcb-render.0.dylib @executable_path/../Resources/lib/libxcb-render.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxcb/lib/libxcb-shm.0.dylib @executable_path/../Resources/lib/libxcb-shm.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo.2.dylib
install_name_tool -change $BREW_PREFIX/opt/pixman/lib/libpixman-1.0.dylib @executable_path/../Resources/lib/libpixman-1.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo.2.dylib
install_name_tool -change $BREW_PREFIX/Cellar/libxcb/1.17.0/lib/libxcb.1.dylib @executable_path/../Resources/lib/libxcb.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libxcb-shm.0.dylib
install_name_tool -change $BREW_PREFIX/opt/libxau/lib/libXau.6.dylib @executable_path/../Resources/lib/libXau.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libxcb-shm.0.dylib
install_name_tool -change $BREW_PREFIX/opt/libxdmcp/lib/libXdmcp.6.dylib @executable_path/../Resources/lib/libXdmcp.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libxcb-shm.0.dylib
install_name_tool -change $BREW_PREFIX/opt/libxau/lib/libXau.6.dylib @executable_path/../Resources/lib/libXau.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libxcb.1.dylib
install_name_tool -change $BREW_PREFIX/opt/libxdmcp/lib/libXdmcp.6.dylib @executable_path/../Resources/lib/libXdmcp.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libxcb.1.dylib
install_name_tool -change $BREW_PREFIX/Cellar/libxcb/1.17.0/lib/libxcb.1.dylib @executable_path/../Resources/lib/libxcb.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libxcb-render.0.dylib
install_name_tool -change $BREW_PREFIX/opt/libxau/lib/libXau.6.dylib @executable_path/../Resources/lib/libXau.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libxcb-render.0.dylib
install_name_tool -change $BREW_PREFIX/opt/libxdmcp/lib/libXdmcp.6.dylib @executable_path/../Resources/lib/libXdmcp.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libxcb-render.0.dylib
install_name_tool -change $BREW_PREFIX/opt/libx11/lib/libX11.6.dylib @executable_path/../Resources/lib/libX11.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libXrender.1.dylib
install_name_tool -change $BREW_PREFIX/opt/libxcb/lib/libxcb.1.dylib @executable_path/../Resources/lib/libxcb.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libX11.6.dylib
install_name_tool -change $BREW_PREFIX/opt/libx11/lib/libX11.6.dylib @executable_path/../Resources/lib/libX11.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libXext.6.dylib
install_name_tool -change $BREW_PREFIX/opt/libpng/lib/libpng16.16.dylib @executable_path/../Resources/lib/libpng16.16.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libfreetype.6.dylib
install_name_tool -change $BREW_PREFIX/opt/freetype/lib/libfreetype.6.dylib @executable_path/../Resources/lib/libfreetype.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libfontconfig.1.dylib
install_name_tool -change $BREW_PREFIX/Cellar/cairo/1.18.0/lib/libcairo.2.dylib @executable_path/../Resources/lib/libcairo.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/gettext/lib/libintl.8.dylib @executable_path/../Resources/lib/libintl.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgobject-2.0.0.dylib @executable_path/../Resources/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libpng/lib/libpng16.16.dylib @executable_path/../Resources/lib/libpng16.16.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/fontconfig/lib/libfontconfig.1.dylib @executable_path/../Resources/lib/libfontconfig.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/freetype/lib/libfreetype.6.dylib @executable_path/../Resources/lib/libfreetype.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libx11/lib/libX11.6.dylib @executable_path/../Resources/lib/libX11.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxext/lib/libXext.6.dylib @executable_path/../Resources/lib/libXext.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxrender/lib/libXrender.1.dylib @executable_path/../Resources/lib/libXrender.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxcb/lib/libxcb.1.dylib @executable_path/../Resources/lib/libxcb.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxcb/lib/libxcb-render.0.dylib @executable_path/../Resources/lib/libxcb-render.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxcb/lib/libxcb-shm.0.dylib @executable_path/../Resources/lib/libxcb-shm.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/pixman/lib/libpixman-1.0.dylib @executable_path/../Resources/lib/libpixman-1.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-gobject.2.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgdk_pixbuf-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/gettext/lib/libintl.8.dylib @executable_path/../Resources/lib/libintl.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgdk_pixbuf-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgobject-2.0.0.dylib @executable_path/../Resources/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgdk_pixbuf-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgmodule-2.0.0.dylib @executable_path/../Resources/lib/libgmodule-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgdk_pixbuf-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgio-2.0.0.dylib @executable_path/../Resources/lib/libgio-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgdk_pixbuf-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/libpng/lib/libpng16.16.dylib @executable_path/../Resources/lib/libpng16.16.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgdk_pixbuf-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/jpeg-turbo/lib/libjpeg.8.dylib @executable_path/../Resources/lib/libjpeg.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgdk_pixbuf-2.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/freetype/lib/libfreetype.6.dylib @executable_path/../Resources/lib/libfreetype.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libharfbuzz.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libharfbuzz.0.dylib
install_name_tool -change $BREW_PREFIX/opt/graphite2/lib/libgraphite2.3.dylib @executable_path/../Resources/lib/libgraphite2.3.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libharfbuzz.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpango-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgobject-2.0.0.dylib @executable_path/../Resources/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpango-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgio-2.0.0.dylib @executable_path/../Resources/lib/libgio-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpango-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/fribidi/lib/libfribidi.0.dylib @executable_path/../Resources/lib/libfribidi.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpango-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/harfbuzz/lib/libharfbuzz.0.dylib @executable_path/../Resources/lib/libharfbuzz.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpango-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/Cellar/pango/1.52.2/lib/libpango-1.0.0.dylib @executable_path/../Resources/lib/libpango-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangocairo-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/Cellar/pango/1.52.2/lib/libpangoft2-1.0.0.dylib @executable_path/../Resources/lib/libpangoft2-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangocairo-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangocairo-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgobject-2.0.0.dylib @executable_path/../Resources/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangocairo-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/harfbuzz/lib/libharfbuzz.0.dylib @executable_path/../Resources/lib/libharfbuzz.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangocairo-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/fontconfig/lib/libfontconfig.1.dylib @executable_path/../Resources/lib/libfontconfig.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangocairo-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/cairo/lib/libcairo.2.dylib @executable_path/../Resources/lib/libcairo.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangocairo-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/Cellar/pango/1.52.2/lib/libpango-1.0.0.dylib @executable_path/../Resources/lib/libpango-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangoft2-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangoft2-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgobject-2.0.0.dylib @executable_path/../Resources/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangoft2-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgio-2.0.0.dylib @executable_path/../Resources/lib/libgio-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangoft2-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/harfbuzz/lib/libharfbuzz.0.dylib @executable_path/../Resources/lib/libharfbuzz.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangoft2-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/fontconfig/lib/libfontconfig.1.dylib @executable_path/../Resources/lib/libfontconfig.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangoft2-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/freetype/lib/libfreetype.6.dylib @executable_path/../Resources/lib/libfreetype.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libpangoft2-1.0.0.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgmodule-2.0.0.dylib @executable_path/../Resources/lib/libgmodule-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/gettext/lib/libintl.8.dylib @executable_path/../Resources/lib/libintl.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgobject-2.0.0.dylib @executable_path/../Resources/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgio-2.0.0.dylib @executable_path/../Resources/lib/libgio-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/pango/lib/libpangocairo-1.0.0.dylib @executable_path/../Resources/lib/libpangocairo-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/pango/lib/libpango-1.0.0.dylib @executable_path/../Resources/lib/libpango-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/harfbuzz/lib/libharfbuzz.0.dylib @executable_path/../Resources/lib/libharfbuzz.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/cairo/lib/libcairo.2.dylib @executable_path/../Resources/lib/libcairo.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/fribidi/lib/libfribidi.0.dylib @executable_path/../Resources/lib/libfribidi.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/cairo/lib/libcairo-gobject.2.dylib @executable_path/../Resources/lib/libcairo-gobject.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/gdk-pixbuf/lib/libgdk_pixbuf-2.0.0.dylib @executable_path/../Resources/lib/libgdk_pixbuf-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/libepoxy/lib/libepoxy.0.dylib @executable_path/../Resources/lib/libepoxy.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/graphene/lib/libgraphene-1.0.0.dylib @executable_path/../Resources/lib/libgraphene-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/pango/lib/libpangoft2-1.0.0.dylib @executable_path/../Resources/lib/libpangoft2-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/fontconfig/lib/libfontconfig.1.dylib @executable_path/../Resources/lib/libfontconfig.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/libpng/lib/libpng16.16.dylib @executable_path/../Resources/lib/libpng16.16.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/libtiff/lib/libtiff.6.dylib @executable_path/../Resources/lib/libtiff.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/jpeg-turbo/lib/libjpeg.8.dylib @executable_path/../Resources/lib/libjpeg.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/opt/cairo/lib/libcairo-script-interpreter.2.dylib @executable_path/../Resources/lib/libcairo-script-interpreter.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libgtk-4.1.dylib
install_name_tool -change $BREW_PREFIX/Cellar/cairo/1.18.0/lib/libcairo.2.dylib @executable_path/../Resources/lib/libcairo.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libpng/lib/libpng16.16.dylib @executable_path/../Resources/lib/libpng16.16.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/fontconfig/lib/libfontconfig.1.dylib @executable_path/../Resources/lib/libfontconfig.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/freetype/lib/libfreetype.6.dylib @executable_path/../Resources/lib/libfreetype.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libx11/lib/libX11.6.dylib @executable_path/../Resources/lib/libX11.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxext/lib/libXext.6.dylib @executable_path/../Resources/lib/libXext.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxrender/lib/libXrender.1.dylib @executable_path/../Resources/lib/libXrender.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxcb/lib/libxcb.1.dylib @executable_path/../Resources/lib/libxcb.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxcb/lib/libxcb-render.0.dylib @executable_path/../Resources/lib/libxcb-render.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/libxcb/lib/libxcb-shm.0.dylib @executable_path/../Resources/lib/libxcb-shm.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/pixman/lib/libpixman-1.0.dylib @executable_path/../Resources/lib/libpixman-1.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/lzo/lib/liblzo2.2.dylib @executable_path/../Resources/lib/liblzo2.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libcairo-script-interpreter.2.dylib
install_name_tool -change $BREW_PREFIX/opt/zstd/lib/libzstd.1.dylib @executable_path/../Resources/lib/libzstd.1.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libtiff.6.dylib
install_name_tool -change $BREW_PREFIX/opt/xz/lib/liblzma.5.dylib @executable_path/../Resources/lib/liblzma.5.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libtiff.6.dylib
install_name_tool -change $BREW_PREFIX/opt/jpeg-turbo/lib/libjpeg.8.dylib @executable_path/../Resources/lib/libjpeg.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/libtiff.6.dylib
install_name_tool -change $BREW_PREFIX/Cellar/librsvg/2.58.1/lib/librsvg-2.2.dylib @executable_path/../Resources/lib/librsvg-2.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/libpng/lib/libpng16.16.dylib @executable_path/../Resources/lib/libpng16.16.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/cairo/lib/libcairo-gobject.2.dylib @executable_path/../Resources/lib/libcairo-gobject.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/freetype/lib/libfreetype.6.dylib @executable_path/../Resources/lib/libfreetype.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/gdk-pixbuf/lib/libgdk_pixbuf-2.0.0.dylib @executable_path/../Resources/lib/libgdk_pixbuf-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgio-2.0.0.dylib @executable_path/../Resources/lib/libgio-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/pango/lib/libpangocairo-1.0.0.dylib @executable_path/../Resources/lib/libpangocairo-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/pango/lib/libpango-1.0.0.dylib @executable_path/../Resources/lib/libpango-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgobject-2.0.0.dylib @executable_path/../Resources/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/gettext/lib/libintl.8.dylib @executable_path/../Resources/lib/libintl.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/harfbuzz/lib/libharfbuzz.0.dylib @executable_path/../Resources/lib/libharfbuzz.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/cairo/lib/libcairo.2.dylib @executable_path/../Resources/lib/libcairo.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so
install_name_tool -change $BREW_PREFIX/opt/libpng/lib/libpng16.16.dylib @executable_path/../Resources/lib/libpng16.16.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
install_name_tool -change $BREW_PREFIX/opt/cairo/lib/libcairo-gobject.2.dylib @executable_path/../Resources/lib/libcairo-gobject.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
install_name_tool -change $BREW_PREFIX/opt/freetype/lib/libfreetype.6.dylib @executable_path/../Resources/lib/libfreetype.6.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
install_name_tool -change $BREW_PREFIX/opt/gdk-pixbuf/lib/libgdk_pixbuf-2.0.0.dylib @executable_path/../Resources/lib/libgdk_pixbuf-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgio-2.0.0.dylib @executable_path/../Resources/lib/libgio-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
install_name_tool -change $BREW_PREFIX/opt/pango/lib/libpangocairo-1.0.0.dylib @executable_path/../Resources/lib/libpangocairo-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
install_name_tool -change $BREW_PREFIX/opt/pango/lib/libpango-1.0.0.dylib @executable_path/../Resources/lib/libpango-1.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libgobject-2.0.0.dylib @executable_path/../Resources/lib/libgobject-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
install_name_tool -change $BREW_PREFIX/opt/glib/lib/libglib-2.0.0.dylib @executable_path/../Resources/lib/libglib-2.0.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
install_name_tool -change $BREW_PREFIX/opt/gettext/lib/libintl.8.dylib @executable_path/../Resources/lib/libintl.8.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
install_name_tool -change $BREW_PREFIX/opt/harfbuzz/lib/libharfbuzz.0.dylib @executable_path/../Resources/lib/libharfbuzz.0.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
install_name_tool -change $BREW_PREFIX/opt/cairo/lib/libcairo.2.dylib @executable_path/../Resources/lib/libcairo.2.dylib $APP_PREFIX/$target/SpaceAcres.app/Contents/Resources/lib/librsvg-2.2.dylib
```

Reply:
I didn't expect for them to be listed explicitly like that. I literally expected something like what I linked above on Windows with a single glob expression. Or maybe something like this (pseudo-code):
```
contents=$APP_PREFIX/$target/SpaceAcres.app/Contents
for path in $BREW_PREFIX/opt/*/lib/*.dylib; do
    cp $path $contents/Resources/lib/$(basename $path)
    install_name_tool -change \
        $(basedir $path) \
        @executable_path/../Resources/lib/$(basename $path) \
        $contents/MacOS/space-acres
done
```

SVG loader will likely remain separate though.